pkgs/development/tools/pyre-check/default.nix

```diff
@@ -0,0 +1,37 @@
+{stdenv, lib, fetchFromGitHub, ocaml, makeWrapper}:
+
+stdenv.mkDerivation (rec {
+
+  name = "pyre-check";
+  version = "0.0.6";
+
+  src = fetchFromGitHub {
+    inherit repo;
+    owner = "facebook";
+    rev = "${version}";
+    sha256 = "0q7f5iin7ilkhs2zgkk527ryw9x1r6pnzgy1lc4d3j3vg6ifhcgc";
+  };
+
+  buildInputs = [ ocaml makeWrapper ];
+
+  buildPhase = ''
+    ./scripts/setup.sh --local
+    make
+  '';
```

Comment:
Generally, we prefer to let the default `buildPhase` do its thing, since it will handle various debugging/parallelization flags, etc.  When we want it to do something non-standard, we do something like this:

```
preBuild = ''
  ./scripts/setup.sh --local  
'';
```
(the default buildPhase will call make for you).  Though in this case it's probably more appropriate to put this as a `preConfigure` hook instead.